content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+        consecutively;
+
+    -   No member intending to serve as Chair or Secretary may be on
+        co-op;
+
+    -   Members wishing to nominate themselves for chair should be a
+        previous WiCS committee member. If no such volunteer exists, the
+        position opens up to new members;
+
+    -   Excluding that of Chair, no positions may be chosen until the
+        first official meeting of the new term's committee;
+
+    -   Existing committee members may compose no more than 75% of the
+        new committee, if more existing members are nominated than 75%
+        can accomodate, each returning member will submit a new
+        application to be reviewed by the serving Director of the WiCS
+        Committee.
+
+4.  If there are any objections to an applicant, this procedure should
+    be followed:
+
+    1.  The objection will be presented to the committee, with
+        sufficient evidence;
+
+    2.  The person being objected to may present a defense;
+
+    3.  The current committee will then take a vote, if a majority of
+        members agree with the objection, the objected person will be
+        denied a position on the committee.
+
+    If the objection is a sensitive concern, it can be brought up to the
+    Chair or Secretary, where they will decide whether to bring the case
+    to the committee and how to proceed. Otherwise, the objected person
+    will be automatically accepted for the next term.
+
+5.  Once the committee has been chosen, the Director of the WiCS
+    Committee must ratify the new committeee.
+
+## Standing Committees ##
+
+-   The Systems Committee (wics-sys) shall:
+
+    1.  Be chaired by the systems administrator;
+
+    2.  Operate all equipment in the possession of the committee;
+
+    3.  Maintain and upgrade the software on the equipment used by the
+        committee.
+
+-   Other Committees\
+    The undergraduate committee can create and task standing committees
+    as necessary. Such standing committees can be dissolved by the
+    undergraduate committee.
+
+## Keys ##
+
+-   Keys will be allocated to the undergraduate committeee as per policy
+    set out in the Memorandum of Understanding with the Mathematics
+    Society;
+
+-   Office Manager and Systems Administrator will both receive keys.
+    Remaining keys will be allocated as the undergraduate committee sees
+    fit.
+
+## Holding Committee Meetings ##
+
+1.  The Secretary is tasked with calling meetings;
+
+2.  Meetings are suggested to be weekly and must occur biweekly;
+
+3.  Meetings must be announced with at least 48 hours of notice;
+
+4.  If the Secretary does not schedule a meeting within a two-week
+    period, any committee member may call a meeting
+
+5.  There will exist two types of meetings, open and closed.
+
+    -   Open Session Meeting - This type of meeting is open to all
+        committee members and guests, by invitation only. In order to
```

Comment:
guests, such as the Director of the WiCS Committee or President of the Computer Science Club,